File: Java/Aksesoris.java
public class Aksesoris extends PetShop {
    private String jenis;
    private String bahan;
    private String warna;
    public Aksesoris() {}
    public Aksesoris(String jenis, String bahan, String warna) {
        this.jenis = jenis;
        this.bahan = bahan;
        this.warna = warna;
    }

    public String get_jenis() {
        return jenis;
    }

    public void set_jenis(String jenis) {
        this.jenis = jenis;
    }

    public String get_bahan() {
        return bahan;
    }

    public void set_bahan(String bahan) {
        this.bahan = bahan;
    }

    public String get_warna() {
        return warna;
    }

    public void set_warna(String warna) {
        this.warna = warna;
    }
}

File: CPP/Aksesoris.cpp
#include "PetShop.cpp"

class Aksesoris : public PetShop
{
private:
    string jenis, bahan, warna;
    
public:
    Aksesoris()
    {
        this->jenis = "";
        this->bahan = "";
        this->warna = "";
    }

    Aksesoris(string jenis, string bahan, string warna)
    {
        this->jenis = jenis;
        this->bahan = bahan;
        this->warna = warna;
    }

    string get_jenis(){return jenis;}
    string get_bahan(){return bahan;}
    string get_warna(){return warna;}

    void set_jenis(string jenis){this->jenis = jenis;}
    void set_bahan(string bahan){this->bahan = bahan;}
    void set_warna(string warna){this->warna = warna;}

    ~Aksesoris()
    {

    }
};